Microsoft.Toolkit.Uwp.UI.Controls/Loading/Loading.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Animation;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Loading control allows to show an loading animation with some xaml in it.
+    /// </summary>
+    public sealed partial class Loading : ContentControl
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Loading"/> class.
+        /// </summary>
+        public Loading()
+        {
+            this.HorizontalContentAlignment = HorizontalAlignment.Stretch;
```

Comment:
Agree, this control has to come with a xaml default template to allow developers customizing it